compiler/rustc_error_messages/locales/en-US/codegen_gcc.ftl

```diff
@@ -0,0 +1,71 @@
+codegen_gcc_ranlib_failure =
+    Ranlib exited with code {$exit_code}
+
+codegen_gcc_layout_size_overflow =
+    {$error}
```

Comment:
This isn't translatable, we should have different slugs for each of the different messages that a layout error can result in.

Reply:
Oops, this is unused so I've removed it.